node/node.go

```diff
@@ -720,9 +788,9 @@ func (n *Node) StartHeartbeat(hbConfig config.HeartbeatConfig, versionNumber str
 	}
 
 	if !n.messenger.HasTopic(HeartbeatTopic) {
-		err := n.messenger.CreateTopic(HeartbeatTopic, true)
-		if err != nil {
-			return err
+		errTopic := n.messenger.CreateTopic(HeartbeatTopic, true)
```

Comment:
delete Topic

Reply:
I changed this in order to remove GoLand's warnings